images/capi/packer/config/ansible-args.json

```
@@ -1,3 +1,5 @@
+  "ansible_scp_extra_args": "-O",
+  "ansible_ssh_args": "-o IdentitiesOnly=yes -o HostKeyAlgorithms=+ssh-rsa -o PubkeyAcceptedAlgorithms=+ssh-rsa"
```

Do we want to use `ansible_common` as a prefix for all variables here?
```suggestion
  "ansible_common_ssh_args": "-o IdentitiesOnly=yes -o HostKeyAlgorithms=+ssh-rsa -o PubkeyAcceptedAlgorithms=+ssh-rsa"
```

---

```json
"ansible_scp_extra_args": "-O",
```
With this I got something like this `ansible-playbook: error: argument --scp-extra-args: expected one argument`
Changing to below works perfectly for GCP
```json
"ansible_scp_extra_args": "\"-O\"",
```

---

Yeah, same happens to me:
```console
$ ansible --version
ansible [core 2.11.5]
  config file = /home/invidian/data/workspaces/clusterapi-flatcar/image-builder/images/capi/ansible.cfg
  configured module search path = ['/home/invidian/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/invidian/.local/lib/python3.10/site-packages/ansible
  ansible collection location = /home/invidian/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/invidian/.local/bin/ansible
  python version = 3.10.5 (main, Jun  6 2022, 18:49:26) [GCC 12.1.0]
  jinja version = 3.1.2
  libyaml = True
```

---

@kopiczko 
```bash
ansible [core 2.12.7]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/aniruddha/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/aniruddha/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.4 (main, Apr  2 2022, 09:04:19) [GCC 11.2.0]
  jinja version = 3.0.3
  libyaml = True
```

---

Added quotations. It also fails for me. I didn't notice in the first place because of my overwrites in my dev branch.